docs/src/main/asciidoc/security.adoc

```diff
@@ -55,35 +55,39 @@ Please see xref:security-built-in-authentication.adoc#mutual-tls[Mutual TLS Auth
 
 === OpenID Connect
 
-`quarkus-oidc` extension provides a reactive, interoperable, multi-tenant enabled OpenID Connect adapter which supports `Bearer Token` and `Authorization Code Flow` authentication mechanisms.
+OpenID Connect (OIDC) is an open authentication protocol that enables client applications to verify the identity of a user when that user tries to connect to a secure server, such as HTTPS.
```

Comment:
I'd like to reword it a bit, and not really involve OAuth2.0 compatibility at this early stage, will try next week :-)